prover/lang/kore-lang.md

```diff
@@ -842,7 +850,25 @@ Simplifications
   rule D inDecls D Ds => true
   rule D inDecls D' Ds => D inDecls Ds
     requires D =/=K D'
-
+```
+Functions `getUniversallyQuantifiedVariables` and `getConclusion`
+assume a pattern of the form:
```

Comment:
These functions should get stuck if the assumption is not satisfied. (Otherwise, it's harder for debugging in the future).

Reply:
These functions do not have any assumptions. A formula without implications or universal quantifiers can be viewed as a conclusion without premises.